### PR TITLE
chore(release): bump to v6.1.0-alpha.0

### PR DESCRIPTION
**Pull request by shlokamin**
# Overview

Bump to 6.1.0 alpha.0

# Changelog

- Bump to 6.1.0-alpha.0

# Review requests

Check changelogs and make sure they seem reasonable. 
@ecormany can you also please update the release notes (both for app-shell and api 😄)

# Risk assessment

Low

### CODE REVIEW

**Review by ecormany — COMMENTED** [review]
Placeholder release notes are fine for alpha 0, especially since this is going into a release branch and not `edge`. Will make a ticket now to update release notes in the release branch in the coming days.

File: api/release-notes.md [hunk 1]
```diff
@@ -4,6 +4,15 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
...
 
+---
+# OT-2 Software Changes in 6.1.0
+
+Welcome to the v6.1.0 release of OT-2 software! This release adds Heater-Shaker support to the Opentrons software ecosystem
+
+## New Features
+
+- Heater-Shaker support
+
```
Comment (sanni-t) [inline api/release-notes.md:15]: Nitpick.
```suggestion

## Known Issues

This alpha is a work in progress! We'll be addressing issues in upcoming releases of 6.1.0.
```

File: app-shell/build/release-notes.md [hunk 2]
```diff
@@ -6,6 +6,16 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
...
+# Opentrons App Changes in 6.1.0
+
+Welcome to the v6.1.0 release of the Opentrons App! This release adds Heater-Shaker support to the Opentrons software ecosystem
+
+## New Features
+
+- Heater-Shaker support
+
+---
+
```
Comment (sanni-t) [inline app-shell/build/release-notes.md:18]:
```suggestion

## Known Issues

This alpha is a work in progress! We'll be addressing issues in upcoming releases of 6.1.0.

```

**Review by sanni-t — APPROVED** [review]
👾

File: app-shell/build/release-notes.md (hunk 2 above)
Comment (ecormany) [reply app-shell/build/release-notes.md:18]: Yeah, good to have that text in there for anyone who stumbles across it.

**Review by mcous — COMMENTED** [review]
LGTM!

**Review by mcous — APPROVED** [review]

**Pull request merged**